builds/jenkins-build-pack-website/Jenkinsfile

```diff
@@ -163,10 +172,21 @@ node()  {
 		stage("Deployment to ${current_environment} Env"){
 			def randomString = utilModule.generateRequestId();
 			def credsId = "jazz-${randomString}";
+			def dataObjValues = getRequiredData();
+			def s3BucketNameValue
```

Comment:
required bucket arn has already been retrieved in line 101/112. Why is it again being fetched?

Reply:
Removed this line and using the already fetched value